dspace-spring-rest/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -40,48 +43,48 @@ public void findAllTest() throws Exception {
         //** GIVEN **
         //1. A community-collection structure with one parent community with sub-community and two collections.
         parentCommunity = CommunityBuilder.createCommunity(context)
-                                          .withName("Parent Community")
-                                          .build();
+            .withName("Parent Community")
+            .build();
```

Comment:
@mspalti : It looks like this PR accidentally performs a lot of code realignment (of existing code).  Is there any chance you can undo the code realignment?  It doesn't seem necessary for the PR, and it distracts from the actual code changes.  My guess is this might have been done automatically by your IDE or something, as the old realignment was perfectly OK by our code style.

Overall, this isn't a big deal, but it'd be a way to make this PR smaller overall.

Reply:
Yes, it was IDE code alignment.  Will fix.